commit 0d0893a74332162397b6a94cbf332a715ba76e0c
Author: Your Name <ivan@malkov.com>
Date:   Thu Aug 18 09:05:55 2022 +0200

    patch2

diff --git a/release/src-rt-5.04axhnd.675x/make.common b/release/src-rt-5.04axhnd.675x/make.common
index de2f353800..b05076154a 100644
--- a/release/src-rt-5.04axhnd.675x/make.common
+++ b/release/src-rt-5.04axhnd.675x/make.common
@@ -804,7 +804,7 @@ else
 BRCM_COMMON_CFLAGS := -Os -fomit-frame-pointer -fno-common -ffixed-r8 -D__ARM_ARCH_7A__ $(BRCM_WERROR_CFLAGS)
 endif
 else
-BRCM_COMMON_CFLAGS := -Os -march=armv7-a -fomit-frame-pointer -mno-thumb-interwork -mabi=aapcs-linux -marm -fno-common -ffixed-r8 -msoft-float -D__ARM_ARCH_7A__ $(BRCM_WERROR_CFLAGS)
+BRCM_COMMON_CFLAGS := -Os -march=armv7-a -mtune=cortex-a7 -mfloat-abi=softfp -mfpu=neon-vfpv4 -fomit-frame-pointer -mno-thumb-interwork -mabi=aapcs-linux -marm -fno-common -ffixed-r8 -D__ARM_ARCH_7A__ $(BRCM_WERROR_CFLAGS)
 endif
 #for mips' -mno-shared (do not generate posistion independent code), arm is -mno-apcs-reentrant which is the default
 ifeq ($(CMS_MEM_LEAK_TRACING),y)
